app/src/main/java/org/fossasia/susi/ai/data/db/DatabaseRepository.kt

```diff
@@ -24,11 +24,10 @@ class DatabaseRepository : IDatabaseRepository {
     var realm: Realm = Realm.getDefaultInstance()
 
     override fun getMessageCount(): Long {
-        val temp = realm.where(ChatMessage::class.java).max(Constant.ID)
-        return if (temp == null)
-            -1
-        else
-            temp as Long
+        val tempNumber = realm.where(ChatMessage::class.java).max(Constant.ID)
+        var tempLong: Long? = null
+        if (tempNumber is Long) tempLong = tempNumber
+        return tempLong ?: -1
```

Comment:
What is tempNumber actually? Int? Double?

Whatever it is, don't cast as Long, properly contruct it using `toLong()`